mock/contracts/asyncChildSC.go

```diff
@@ -26,15 +28,21 @@ func transferToThirdPartyAsyncChildMock(instanceMock *mock.InstanceMock, config
 			return instance
 		}
 
-		err := handleChildBehaviorArgument(host, big.NewInt(0).SetBytes(arguments[2]))
+		var behavior byte
+		if len(arguments[2]) != 0 {
+			behavior = arguments[2][0]
+		} else {
+			behavior = 0
```

Comment:
Optional: replace `var behavior byte` with `behavior := byte(0)`, then remove the `else` branch. It is equivalent.

Reply:
done